godot-ffi/src/linux.rs

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) godot-rust; Bromeon and contributors.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+//! Linux-specific configuration.
```

Comment:
Maybe this file could be called `linux_reload_workaround.rs` to be more specific. If there is other Linux-specific logic in the future, we can always generalize or add a dedicated `linux` module.